app/order/components/OrderView.tsx

```diff
@@ -1,35 +1,21 @@
-import { useQuery } from 'blitz'
+import { Order, OrderItemSnapshot, OrderStatus, Shop } from '@prisma/client'
 import { useMemo } from 'react'
 import { FiTruck } from 'react-icons/fi'
 
 import { EmptyState } from 'app/core/components/EmptyState'
 
-import getOrders from '../queries/getOrders'
 import { OrderCard } from './OrderCard'
 
 export interface OrderViewProps {
-  filter: number
+  filter: OrderStatus
+  orders: (Order & { shop: Shop; items: OrderItemSnapshot[] })[]
 }
 
-export function OrderView({ filter }: OrderViewProps) {
-  const status = useMemo(() => {
-    switch (filter) {
-      case 1:
-        return 'PENDING'
-      case 2:
-        return 'SHIPPED'
-      case 3:
-        return 'COMPLETED'
-      case 4:
-        return 'CANCELLED'
-      default:
-        return 'PENDING'
-    }
-  }, [filter])
-  const [orders] = useQuery(getOrders, {
-    status: status,
-  })
-  if (orders.length == 0) {
+export function OrderView({ filter, orders }: OrderViewProps) {
+  const filtered_orders = useMemo(() => {
```

Comment:
Use camel case naming

Reply:
Also it would be better to do the filtering in the orders page itself, so this component doesn't have to be aware of the status filter.